package.json

```diff
@@ -74,27 +74,27 @@
     "@storybook/addon-knobs": "^3.4.11",
     "@storybook/addon-links": "^3.4.11",
     "@storybook/react": "^3.4.11",
-    "autoprefixer": "^9.1.5",
+    "autoprefixer": "^9.4.3",
     "babel-core": "^6.26.3",
-    "babel-jest": "^23.4.2",
+    "babel-jest": "^23.6.0",
```

Comment:
Support for `babel.config.js` (see [changelog](https://github.com/facebook/jest/blob/master/CHANGELOG.md))